clusterloader2/testing/load/service.yaml

```diff
@@ -0,0 +1,10 @@
+kind: Service
```

Comment:
nit: Can you move the kind below the apiVersion?

Reply:
Done.